Sources/Widgets/Widgets3D/EllipseWidget/state.js

```diff
@@ -19,7 +19,7 @@ export default function generateState() {
       name: 'point2Handle',
       initialValues: {
         scale1: 10,
-        origin: [0, 0, 0],
+        origin: [undefined, undefined, undefined],
```

Comment:
What's the purpose of setting these to undefined?

Reply:
it is a way to have the point2 handle not initialized hence not visible (but with visible property set to true).
For some reason, I can't set it to `null`, `undefined` or `[]`.

Reply:
I find this approach opaque as to its purpose. Is it not sufficient to toggle the visible property as needed?

Reply:
handles used to not visible by default and I got feedback from developers confused with widget initialization. For example, someone wanted to initialize the spline widget programatically, he was doing :
```
points.forEach((point) => {
      widgetFactory.getWidgetState().addHandle().setOrigin(...point);
    });
```
where he should have done
```
points.forEach((point) => {
      const handle = widgetFactory.getWidgetState().addHandle();
      handle.setOrigin(...point);
      handle.setVisible(true);
    });
```

When we create a prop, we expect it to be visible by default.

Moreover, when dealing with handles in behavior, if we keep `[0, 0, 0]` as default value, it is hard to know if the handle has yet been initialized or not.
WDYT ?

Reply:
Ah, fair point. I think having a null/[] origin can effectively mean "do not show". That also means representations must be changed to support this, e.g. SphereHandleRepresentation should see that origin is null or empty and not show that state.